python/astro_metadata_translator/bin/translateheader.py

```diff
@@ -328,10 +344,19 @@ def main():
     output_mode = args.mode
     if args.quiet:
         output_mode = "none"
+    elif args.dumphdr:
+        output_mode = "yaml"
```

Comment:
Shouldn't the default be set at construction time of the argument parser rather than in code later?

Reply:
Can I make the argument parser set another argument? ie make --mode be set to "yaml" if --dumphdr is true?